app/templates/base.html

```diff
@@ -21,97 +21,101 @@
+   <link rel="stylesheet" type="text/css" src="{{ url_for('static', filename='browse.css') }}" />
    <!-- custom css stylesheet end -->
   {% assets "css" %}<link href="{{ ASSET_URL }}" rel="stylesheet">{% endassets %}
 {% endblock %}
 
 {% block bodyStart %}
-  {% if "cookies_policy" not in request.cookies %}
-    {% set html %}
-      <p class="govuk-body">We use some essential cookies to make this service work.</p>
-      <p class="govuk-body">We’d like to set additional cookies so we can remember your settings, understand how people use the service and make improvements.</p>
-    {% endset %}
-
-    {% set acceptHtml %}
-      <p class="govuk-body">You’ve accepted additional cookies. You can <a class="govuk-link" href="{{ url_for('main.cookies') }}">change your cookie settings</a> at any time.</p>
-    {% endset %}
-
-    {% set rejectHtml %}
-      <p class="govuk-body">You’ve rejected additional cookies. You can <a class="govuk-link" href="{{ url_for('main.cookies') }}">change your cookie settings</a> at any time.</p>
-    {% endset %}
-
-    {{ govukCookieBanner({
-      'ariaLabel': "Cookies on " + config['SERVICE_NAME'],
-      'attributes': {
-        'id': "cookie-banner"
-      },
-      'messages': [
-        {
-          'attributes': {
-            'id': "default-message"
-          },
-          'headingText': "Cookies on " + config['SERVICE_NAME'],
-          'html': html,
-          'actions': [
-            {
-              'attributes': {
-                'id': "accept-cookies"
-              },
-              'text': "Accept additional cookies",
-              'type': "button",
-              'name': "cookies",
-              'value': "accept"
-            },
-            {
-              'attributes': {
-                'id': "reject-cookies"
-              },
-              'text': "Reject additional cookies",
-              'type': "button",
-              'name': "cookies",
-              'value': "reject"
-            },
-            {
-              'text': "View cookies",
-              'href': url_for('main.cookies')
-            }
-          ]
-        },
-        {
-          'attributes': {
-            'id': "accepted-message"
-          },
-          'html': acceptHtml,
-          'role': "alert",
-          'hidden': true,
-          'actions': [
-            {
-              'attributes': {
-                'id': "accepted-hide"
-              },
-              'text': "Hide this message"
-            }
-          ]
-        },
-        {
-          'attributes': {
-            'id': "rejected-message"
-          },
-          'html': rejectHtml,
-          'role': "alert",
-          'hidden': true,
-          'actions': [
-            {
-              'attributes': {
-                'id': "rejected-hide"
-              },
-              'text': "Hide this message"
-            }
-          ]
-        }
-      ]
-    }) }}
-  {% endif %}
+{% if "cookies_policy" not in request.cookies %}
```

Comment:
This file has become arguably less readable now the indentation has been removed. Was this an autoformatter?

Reply:
Reverted formatting, looks like VS Code is causing an issue here. Should be cleaner to view now.